os/drivers/lcd/mipi_lcd.c

```diff
@@ -348,6 +371,17 @@ FAR struct lcd_dev_s *mipi_lcdinitialize(FAR struct mipi_dsi_device *dsi, struct
 		lcddbg("ERROR: LCD Init sequence failed\n");
 	}
 	priv->config->backlight(CONFIG_LCD_MAXPOWER);
-
+#ifdef CONFIG_LCD_ST7701SN
+	for (int itr = 0; itr < NUM_OF_LCD_BUFFER; itr++) {
+		lcd_buffer[itr] = (uint8_t *)malloc(CONFIG_LCD_XRES * CONFIG_LCD_YRES * 2 + 1);
```

Comment:
This should be `kmm_malloc` to get memory at kernel side. (Because the driver is in kernel context)
To use kmm_malloc, you should include <tinyara/kmalloc.h>